src/development/platform-integration/platform-views.md

```diff
@@ -137,7 +137,7 @@ import 'package:flutter/services.dart';
 ```dart
```

Comment:
This is not the source of truth for this content. It is copied in from `lib/platform_views/native_view_example_2.dart` so you'll need to edit it there and then use the tooling to update this file.

@khanhnwin can supply more detail on how to run the snippet tooling.

Reply:
Hi! Thanks for reminding me, I'll have to add instructions for the excerpts tooling to our readme. As @domesticmouse mentioned - once the Dart file with the code that is being copied in has changed, from the root of flutter/website you'll want to run the code excerpts script like so: `./tool/refresh-code-excerpts.sh` Let me know if you run into any issues.